Python/Instagram_BOT/README.md

```diff
@@ -0,0 +1,45 @@
+# 🍕 Instagram BOT - [Harshit Bansal](https://github.com/harshitbansall)
+## Prerequisites
+* Create Virtual Environment
+```
+python -m venv env
+```
+* Activate the Environment
+```
+env\Scripts\activate
+```
+* Install all the dependencies
+```
+pip install -r requirements.txt
+```
+## Usage
+* Start the Script <code> Instragram BOT.py</code>
+<img src = "https://user-images.githubusercontent.com/24206423/156173269-b6a641b0-6704-4fde-ac56-02a9f2763938.png" height = 300>
+
+
+* <code> 1</code> to Download Profile Image from Username
+* <code> 2</code> to Download All Posts from Username
+* <code> 3</code> to Download Images from Hashtags
+
+## Examples
+* <code>Example 1</code> - Download Profile Image of <code>jacquelinef143</code>
+<div>
+<img src = "https://user-images.githubusercontent.com/24206423/156174852-2615dd80-548e-4839-8105-a0accdefba6f.png" height = 200>
+<img src = "https://user-images.githubusercontent.com/24206423/156174856-1bd8eb95-69da-4591-814f-b75cde6db9e6.png">
+</div>
+<br>
+
+* <code>Example 2</code> - Download Post Images of <code>jacquelinef143</code>
+
+<div align = "center">
+<img src = "https://user-images.githubusercontent.com/24206423/156177713-f4b8a664-fc85-42fc-8d8f-c72b76442dde.gif">
+</div>
+<br>
+
+* <code>Example 3</code> - Download Hash Tag Images of <code>dogs</code>
+
+<div align = "center">
+<img src = "https://user-images.githubusercontent.com/24206423/156178673-6dc13f7b-49d6-4284-8cbd-9310356ad300.gif">
+</div>
+
+### If you liked this script and and used it, Please follow [Harshit Bansal](https://github.com/harshitbansall).
```

Comment:
Just add an author section with a link to your profile. If someone is looking to follow, they would do it without being prompted.